### PR TITLE
test/purge: assert database state after purging

### PR DESCRIPTION
**Pull request by alxndrsn**

### CODE REVIEW

**Comment by alxndrsn** [comment]
Perhaps more idiomatic to move assertion functions into `lib/model/query/forms.js`

**Review by ktuite — APPROVED** [review]

**Comment by alxndrsn** [comment]
@ktuite is this preferred to #1179?

File: test/integration/task/purge.js [hunk 1]
```diff
@@ -8,144 +10,194 @@ const testData = require('../../data/xml');
 // of purged forms, but the full functionality is more thoroughly tested in
 // test/integration/other/form-purging.js
 
+const decorate = container => {
```
Comment (alxndrsn) [inline test/integration/task/purge.js:13]: rename to `withDeleteAssertions`

**Comment by matthew-white** [comment]
@alxndrsn, I just wanted to check about the status of this PR. I see that the PR is approved. Is it ready to merge? Or are you waiting for feedback from @ktuite?

**Comment by alxndrsn** [comment]
@matthew-white I think this is ready to merge, unless you have reservations.

**Pull request merged**